third_party/terraform/tests/resource_monitoring_slo_test.go

```diff
@@ -104,6 +104,29 @@ func TestAccMonitoringSlo_basic(t *testing.T) {
 	})
 }
 
+func TestAccMonitoringSlo_availabilitySli(t *testing.T) {
+	t.Parallel()
+
+	var generatedId string
+	vcrTest(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckMonitoringSloDestroyProducer(t),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccMonitoringSlo_availabilitySli(),
```

Comment:
Can you test removing the value as well? It may take an unusual update request to do so, I'm not sure!